WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/RangedDatePicker.swift

```diff
@@ -7,6 +7,35 @@ protocol RangedDateTextFormatter {
     func format(start: Date, end: Date) -> String
 }
 
+/// Hosting controller for `RangedDatePicker`
+///
+final class RangedDatePickerHostingController: UIHostingController<RangedDatePicker> {
+    init(startDate: Date? = nil,
+         endDate: Date? = nil,
+         datesFormatter: RangedDateTextFormatter,
+         customApplyButtonTitle: String? = nil,
+         datesSelected: ((_ start: Date, _ end: Date) -> Void)? = nil) {
+        super.init(rootView: RangedDatePicker(startDate: startDate,
+                                              endDate: endDate,
+                                              datesFormatter: datesFormatter,
+                                              customApplyButtonTitle: customApplyButtonTitle,
+                                              datesSelected: datesSelected))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: Constant
+
+private extension RangedDatePickerHostingController {
+    enum Localization {
+        static let apply = NSLocalizedString("Apply", comment: "Apply navigation button in custom range date picker")
+    }
+}
```

Comment:
We should delete this as we don't use this anymore.

Reply:
Good catch, thank you. Done in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/ca820bf0d3a7a47fa12390d4be9271bb97ff5a14